dlt/load/configuration.py

```diff
@@ -20,4 +24,6 @@ class LoaderConfiguration(PoolRunnerConfiguration):
     truncate_staging_dataset: bool = False
 
     def on_resolved(self) -> None:
-        self.pool_type = "none" if self.workers == 1 else "thread"
+        self.pool_type = (
```

Comment:
good. must be None (no threads spawned)

Reply:
I'm not sure what you mean? Should I change something here?